examples/worlds/camera_sensor.sdf

```diff
@@ -8,127 +8,34 @@
       <max_step_size>0.001</max_step_size>
       <real_time_factor>1.0</real_time_factor>
     </physics>
-    <plugin
-      filename="gz-sim-physics-system"
-      name="gz::sim::systems::Physics">
-    </plugin>
     <plugin
       filename="gz-sim-sensors-system"
       name="gz::sim::systems::Sensors">
       <render_engine>ogre2</render_engine>
     </plugin>
-    <plugin
-      filename="gz-sim-user-commands-system"
-      name="gz::sim::systems::UserCommands">
-    </plugin>
-    <plugin
-      filename="gz-sim-scene-broadcaster-system"
-      name="gz::sim::systems::SceneBroadcaster">
-    </plugin>
 
     <scene>
       <ambient>1.0 1.0 1.0</ambient>
       <background>0.8 0.8 0.8</background>
       <grid>true</grid>
     </scene>
 
+    <gz:policies>
+      <include_default_gui_plugins>true</include_default_gui_plugins>
```

Comment:
```suggestion
      < include_gui_default_plugins >true</include_gui_default_plugins >
```

Reply:
Fixed in https://github.com/gazebosim/gz-sim/pull/2497/commits/303e18ba127b39c75d6c9873d26a7d06f266e7f6